scenario.py

```diff
@@ -55,9 +57,16 @@ def __init__(self, params, experiment_path):
 
         # For configuring if data samples are split between partners randomly or in a stratified way...
         # ... so that they cover distinct areas of the samples space
-        self.samples_split_option = params[
-            "samples_split_option"
-        ]  # Toggle between 'random' and 'stratified'
+        if "samples_split_option" in params:
+            self.samples_split_option = params["samples_split_option"]
+        else:
+            self.samples_split_option = "random"
+        if "advanced_split" in params:
+            self.is_advanced_split = True
+            self.advanced_split = params["advanced_split"]
+        else:
+            self.is_advanced_split = False
+            self.advanced_split = "No advanced split provided"
```

Comment:
What is the use of this line ?
I would remove it

Reply:
Done